### PR TITLE
fall back to default language in case translation missing

### PR DESCRIPTION
**Pull request by mkangia**
https://manage.dimagi.com/default.asp?260866

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -1457,7 +1457,8 @@ def audio_by_language(self, lang, strict=False):
     def custom_icon_form_and_text_by_language(self, lang):
         custom_icon = self.custom_icon
         if custom_icon:
-            return custom_icon.form, custom_icon.text[lang]
+            custom_icon_text = custom_icon.text.get(lang) or custom_icon.text.get(self.get_app().default_language)
```
Comment (orangejenny) [inline corehq/apps/app_manager/models.py]: could be more concise: `custom_icon.text.get(lang, self.get_app().default_language)`

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/app_manager/models.py]: Is `""` valid icon text?  If so, this could be potentially confusing, as the empty string is falsy.

**Review by esoergel — APPROVED** [review]

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (esoergel) [reply corehq/apps/app_manager/models.py]: I think you'd want this:
```python
custom_icon.text.get(lang, custom_icon.text.get(self.get_app().default_language))
```

Comment (orangejenny) [reply corehq/apps/app_manager/models.py]: oops, i was sloppy - thanks Ethan :)

Comment (mkangia) [reply corehq/apps/app_manager/models.py]: Actually did the opposite to reduce nesting.

Comment (mkangia) [reply corehq/apps/app_manager/models.py]: looks like the change `custom_icon.text.get(lang, custom_icon.text.get(self.get_app().default_language))` would take care of it. Will make that change.

**Pull request merged**